tests/snapshots/test_es_publish_pipeline.ambr

```diff
@@ -11,6 +11,78 @@
     'hass.entity.attributes': dict({
       'bool': True,
       'collision_test': 'second value',
+      'complex_dict': dict({
```

Comment:
> Was the old behavior that it would produce a stringified json as the value?

Ohhhhh I bet you're right, I forgot about that. We should restore that behavior

Reply:
Thanks for catching this